RLBotDiscordBot/calendars.py

```diff
@@ -0,0 +1,156 @@
+import requests
+import datetime
+import json
+import logging
+import sys
+import random as r
+from urllib.parse import urlparse
+import re
+import os
+from config import GOOGLE_API_KEY
+
+import discord
+from discord.ext import commands
+from discord.utils import get
+
+
+async def checkCalendar(message):
+    load_dotenv()
+    api_key = GOOGLE_API_KEY
+    args = message.content.split(" ")
+    can_run = False
+    for i in args:
+        if i == "!tournaments":
+            can_run = True
+    if can_run:
+        months = ["January", "February", "March", "April", "May", "June", "July", "August", "September","October", "November", "December"]
+        today = str(datetime.datetime.now())
+        checkDay = today.split(" ")[0] + "T"
+        checkTime = today.split(" ")[1]
+        checkTime = checkTime.split(":")
+        checkTime[2] = str(int(float(checkTime[2])))
+        checkTime[2] = checkTime[2] + "0Z" if checkTime[2] == "0" else checkTime[2] + "Z"
+        checkTime = ":".join(checkTime)
+        toCheck = checkDay + checkTime
+        r = requests.get(f"https://www.googleapis.com/calendar/v3/calendars/rlbotofficial@gmail.com/events?maxResults=3&timeMin={toCheck}&key={api_key}")
```

Comment:
I want to change my suggestion:
```python
FORMAT = "%Y-%m-%dT%H:%M:%SZ"
today = datetime.datetime.utcnow()
to_check = today.strftime(FORMAT)

# And elsewhere, when parsing the response from google:
some_date = datetime.datetime.strptime("2013-07-12T07:00:00Z", FORMAT)
```
This is basically what Redox already suggested.

And for printing out stuff like "13 days from now", this might be handy:

https://natural.readthedocs.io/en/latest/natural/date.html